src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: function(num) {
+      this.result += num;
+    },
+
+    subtract: function(num) {
+      this.result -= num;
+    },
```

Comment:
you can do it just like that
```suggestion
    add(num) {
      this.result += num;
    },

    subtract(num) {
      this.result -= num;
    },
```